CDT/include/Triangulation.h

```diff
@@ -743,6 +781,12 @@ class CDT_EXPORT Triangulation
     IntersectingConstraintEdges::Enum m_intersectingEdgesStrategy;
     T m_minDistToConstraintEdge;
     TriIndVec m_vertTris; /// one triangle adjacent to each vertex
+
+    // C++11 required for std::fucntion callbacks
+    #ifndef CDT_CALLBACK_DISABLED    
+    std::function<void(TriInd, CallbackType)> callback_tri; // callback function for triangle creation
+    std::function<void(VertInd, CallbackType)> callback_vertex; // callback function for vertex insertation
```

Comment:
It seems Triangulation class members follow `m_memberName` naming style. 
Naming style of `callback_tri` and `callback_vertex` make members inconsistent.